### PR TITLE
New game over screens

### PR DESCRIPTION
**Pull request by AbhiKapoor55**
- Added New Game Over Screen based on Rodrigo's design 
- Added New Win Screen based on Rodrigo's design

### CODE REVIEW

**Review by slakh96 — CHANGES_REQUESTED** [review]
Minor nitpick: Win screen text is a bit wordy, is it possible to change it? 
image

**Review by slakh96 — CHANGES_REQUESTED** [review]
oi theres an "alienfacceRed.png.meta" in this MR 😱

**Review by gazijarin — APPROVED** [review]
lgtm je suis par favor

**Review by slakh96 — APPROVED** [review]
everything works
